src/Opdex.Platform.WebApi/Controllers/LiquidityPoolsController.cs

```diff
@@ -43,8 +41,14 @@ public LiquidityPoolsController(IMediator mediator, IMapper mapper, IApplication
     /// <param name="filters">Liquidity pool filter options.</param>
     /// <param name="cancellationToken">Cancellation token.</param>
     /// <returns><see cref="LiquidityPoolsResponseModel"/> of matching results and paging details.</returns>
+    /// <response code="200">Liquidity pool results returned.</response>
+    /// <response code="400">The request is not valid.</response>
+    /// <response code="401">Unauthorized.</response>
+    /// <response code="429">Too many requests.</response>
```

Comment:
Just a thought, wonder if should add 500 to all endpoints with 'Unexpected Error', not for this PR really just in general.

Reply:
I'd say so, I'll add them

Reply:
I've added a global operation processor which will apply 500 response documentation to all endpoints. Replaced 429 documentation with a global operation processor as well.